Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -1078,11 +1078,25 @@ - (void)_didCommitLayerTree:(const WebKit::RemoteLayerTreeTransaction&)layerTree
         }
     }
 
+#if ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION) || ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
+    [self _removeScrollingNodesFromCoordinator:layerTreeTransaction.destroyedLayers()];
 #if ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION)
     [self _updateOverlayRegions:layerTreeTransaction.changedLayerProperties() destroyedLayers:layerTreeTransaction.destroyedLayers()];
+#endif // ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION)
 #endif
 }
 
+#if ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION) || ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
+- (void)_removeScrollingNodesFromCoordinator:(const Vector<WebCore::GraphicsLayer::PlatformLayerID>&)destroyedLayers
```

Comment:
The function name says "nodes" but the arguments are layers. And the underlying data is a set of `GraphicsLayer::PlatformLayerID`.

This would be clearer if:
1. m_fixedScrollingNodeLayerIDs had "interaction region" in the name
2. Don't say "node layerIDs", just "fixed layers"
3. We normally use "viewport constrained" as a term that includes sticky and fixed.